### PR TITLE
Channel view layout issue 

### LINKED ISSUE #1499: Channel elements at top do not organize when window is smaller
**Behavior of the program**
Channel info elements (header) overlap and generally dont change to look nice for smaller window size.
There is also a random large gap between the top and the info.

**Expected behavior**
To be sorted with no overlapping or gaps

**To Reproduce**
1. Go on any channel
2. Change window size to small

**Screenshots**
image

 - FreeTube version: Latest

### PR DESCRIPTION
**Pull request by JInfantesC**
---
Channel view layout issue 
---

**Pull Request Type**
- [X] Bugfix
- [ ] Feature Implementation

**Related issue**
Closes #1499

**Description**
This request modifies the CSS and HTML layout the channel view

**Screenshots (if appropriate)**

**Testing (for code that is not small enough to be easily understandable)**
Only CSS and HTML changes.

**Desktop (please complete the following information):**
 - OS: Ubuntu Linux
 - OS Version: 21.04
 - FreeTube version: Release 0.13.2 Beta

### CODE REVIEW

**Review by peepopoggers — APPROVED** [review]

**Review by ChunkyProgrammer — CHANGES_REQUESTED** [review]

File: src/renderer/views/Channel/Channel.vue [hunk 1]
```diff
@@ -19,34 +19,44 @@
...
+              </span>
+            </div>
+          </div>
+
+          <ft-button
+            :label="subscribedText"
+            background-color="var(--primary-color)"
+            text-color="var(--text-with-main-color)"
+            class="subscribeButton"
+            style="align-self: center"
```
[the remark this answers is not in the document]
Comment (peepopoggers) [reply src/renderer/views/Channel/Channel.vue]: Fixed

File: src/renderer/views/Channel/Channel.vue [hunk 2]
```diff
@@ -19,34 +19,44 @@
...
+      >
+        <div
+          class="channelInfo"
+        >
+          <div style="display:flex">
+            <img
+              class="channelThumbnail"
+              :src="thumbnailUrl"
+            >
+            <div style="display: flex;justify-content: center;flex-direction: column;padding-left: 1em;">
```
[the remark this answers is not in the document]
Comment (peepopoggers) [reply src/renderer/views/Channel/Channel.vue]: Fixed

File: src/renderer/views/Channel/Channel.vue [hunk 3]
```diff
@@ -19,34 +19,44 @@
...
-            <span v-if="subCount === 1">{{ $t("Channel.Subscriber") }}</span>
-            <span v-else>{{ $t("Channel.Subscribers") }}</span>
-          </span>
+      <div
+        class="channelInfoContainer"
+      >
+        <div
+          class="channelInfo"
+        >
+          <div style="display:flex">
```
[the remark this answers is not in the document]
Comment (peepopoggers) [reply src/renderer/views/Channel/Channel.vue]: Fixed

**Review by peepopoggers — APPROVED** [review]
Made a few edits and is approved.

**Review by ChunkyProgrammer — APPROVED** [review]

**Comment by ChunkyProgrammer** [comment]
LGTM

**Review by peepopoggers — APPROVED** [review]

**Review by jasonhenriquez — APPROVED** [review]
LGTM!

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by peepopoggers — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Review by PrestonN — APPROVED** [review]

**Pull request merged**